tests/unit/test_cromshellconfig.py

```diff
@@ -0,0 +1,125 @@
+from cromshell.utilities import cromshellconfig
+from importlib import reload
+from pathlib import Path
+import pytest
+import os
+
+
+class TestCromshellConfig:
+    """Test the cromshell config functions and variables"""
+
+    def test_override_slim_metadata_parameters(self):
+        reload(cromshellconfig)
+        default_slim_parameters = (
```

Comment:
There should be a way to place this in the cromshell_config object and read it from the module directly.
This way we avoid the potential problem of having two default values are sometime later the two go out of sync.

Reply:
After taking a closer look, line 18 pulls in the cromshell_config object directly and this assert statement is to check whether there's been any changes to the default. If I did make the change to pull in the variable value from the module directly in this line then line 18 would just be checking two of the same variables. I could remove this test function if its not useful?

Reply:
or maybe just check that its not empty

Reply:
Yeah, I think a null/empty check should be good enough.